tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php

```diff
@@ -305,6 +305,67 @@ class#
             ['space_before_parenthesis' => true, 'inline_constructor_arguments' => false],
         ];
 
+        yield 'single attribute on separate line' => [
+            <<<'EOF'
+                <?php
+                $a = new
+                #[FOO]
+                class() {};
+                EOF,
+        ];
+
+        yield 'multiple attributes on separate line' => [
+            <<<'EOF'
+                <?php
+                $a = new
+                #[FOO]
+                #[\Ns\Bar]
```

Comment:
[This is valid code](https://3v4l.org/70gJp) that is handled by this fixer currently, but maybe we could include it in test case to avoid regression?

```suggestion
                #[FOO] // Added because ...
                #[\Ns\Bar]
```